spec/rspec/mocks/at_most_spec.rb

```diff
@@ -79,6 +79,7 @@ module Mocks
         expect {
           @double.do_something
         }.to raise_error(/expected: at most 2 times.*received: 3 times/m)
+        reset @double
```

Comment:
So this `expect { }.to raise_error; reset double` pattern is showing up over, and over and over again through the spec suite.  I'm thinking we should come up with a common abstraction to handle that.

WDYT of this?

``` ruby
expect_fast_failure_from(double, /expected: at most 2 times.*received: 3 times/m) do
  double.do_something
end
```

`expect_fast_failure_from` would take of `expect { }.to raise_error` and the `reset`.